src/main.jsx

```diff
@@ -18,6 +20,7 @@ ReactDOM.createRoot(document.getElementById('root')).render(
       <Global styles={GlobalStyles} />
       <ErrorBoundary>
         <App />
+        <ToastContainer />
```

Comment:
it will works fine without declaration of container, so you can just remove it from there

Reply:
removed it